### PR TITLE
Add support for COPY command new in Redis 6.2

### PR DESCRIPTION
**Pull request by malinaa96**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `$ tox` pass with this change (including linting)?
- [ ] Does travis tests pass with this change (enable it first in your forked repo and wait for the travis build to finish)?
- [x] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

Add support for COPY command available since Redis 6.2 as requested in https://github.com/andymccurdy/redis-py/issues/1434.

### CODE REVIEW

File: redis/client.py [hunk 1]
```diff
@@ -1612,6 +1612,24 @@ def bitpos(self, key, bit, start=None, end=None):
...
+
+        ``destination_db`` an alternative destination database. By default, 
+        the ``destination`` key is created in the source Redis database.
+
+        ``replace`` whether the ``destination`` key should be removed before
+        copying the value to it. By default, the value is not copied if
+        the ``destination`` key already exists.
+        """
+        params = [source, destination]
+        if destination_db:
```
Comment (chayim) [inline redis/client.py:1627]: Can you change if change to [PEP-8](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html)'s ```if destination_db is None```

**Comment by chayim** [comment]
@malinaa96 Mind merging the latest master? Unit tests pass there, and we can see this all the way through - with your changes.

File: redis/client.py (hunk 1 above)
Comment (malinaa96) [reply redis/client.py:1627]: Sure, no problem

**Comment by chayim** [comment]
LGTM. Thanks @malinaa96 !

**Review by chayim — APPROVED** [review]

**Pull request merged**